packages/theming/atlas/src/themesource/atlas_core/web/design-properties.json

```diff
@@ -173,8 +173,8 @@
             ]
         },
         {
-            "name": "Align self",
-            "oldNames": ["Align Self"],
+            "name": "Move to",
+            "oldNames": ["Align self"],
```

Comment:
Camel cased “Self” should also be added for compatibility reasons

Reply:
Please never resolve a conversation in our repo.

Reply:
Please be aware the reviewer is the responsible to resolve discussions.

Reply:
aah oke, sorry, don't know how everything works, figuring out while doing :x